src/types/request.rs

```diff
@@ -256,25 +374,98 @@ impl PyRequest {
 
     pub fn json(&self, py: Python) -> PyResult<PyObject> {
         match self.body.as_ref(py).downcast::<PyString>() {
-            Ok(python_string) => match serde_json::from_str(python_string.extract()?) {
-                Ok(Value::Object(map)) => {
-                    let dict = PyDict::new(py);
-
-                    for (key, value) in map.iter() {
-                        let py_key = key.to_string().into_py(py);
-                        let py_value = match value {
-                            Value::String(s) => s.as_str().into_py(py),
-                            _ => value.to_string().into_py(py),
-                        };
-
-                        dict.set_item(py_key, py_value)?;
-                    }
+            // use python orjson deserialization
+            // let orjson_mod = PyModule::import(py, "orjson")?;
+            // let fun = orjson_mod.getattr("loads")?;
+            // let parsed_json = fun.call1((python_string,))?;
```

Comment:
@tqq1994516 , do we need this?

Reply:
> do we need this?

It shouldn't be necessary. I tried to directly convert Python objects using python `orjson` at that time. Later, I used rust `serde` for the conversion, so I commented out this part of the logic.